framework/Samplers/MCMC/MCMC.py

```diff
@@ -367,3 +451,26 @@ def _addToSolutionExport(self, rlz):
     if self._burnIn < self.counter:
       rlz = dict((var, np.atleast_1d(val)) for var, val in rlz.items())
       self._solutionExport.addRealization(rlz)
+
+  @staticmethod
+  def tuneScalingParam(scale, acceptRate):
+    """
+      Tune the scaling parameter for the proposal distribution
+      Borrowed from PyMC3
```

Comment:
do we have a reference for these suggested scaling multiplier?

Reply:
I do not find the reference. It seems to me the scaling is based on experience.